TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -37,7 +37,7 @@ public class Workspace
         private SymbolTable _customSymbols;
         private string _rootDirectoryFullName;
         private string _workspaceName;
-        private string[] _extensions = { ".cbl", ".cpy" };
+        private string[] _extensions = { ".cbl", ".cpy", ".copy" };
```

Comment:
Can be replaced by Helpers.DEFAULT_EXTENSIONS

Reply:
done.